### PR TITLE
Updated match_tensor

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

Issue link:#1490

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/references/pysdk_api_reference.md [hunk 1]
```diff
@@ -1706,38 +1706,49 @@ for question in questions:
...
 
 ```python
-table_object.match_tensor(vector_column_name, tensor_data, tensor_data_type, method_type, topn, extra_option)
+table_object.match_tensor(vector_column_name, tensor_data, tensor_data_type, method_type, extra_option)
 ```
 
-Builds a KNN tensor search expression. Find the top n closest rows to the given tensor according to chosen method.
-
-For example, find k most match tensors generated by ColBERT.
+Performs a k-nearest neighbor (KNN) tensor search on a specified column to identify the top n closest rows to the given tensor. Suitable for working with tensors generated by ColBERT.
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:1712]: n -> k

Comment (JinHai-CN) [reply docs/references/pysdk_api_reference.md:1712]: remove: Suitable for working with tensors generated by ColBERT.

File: docs/references/pysdk_api_reference.md [hunk 2]
```diff
@@ -1706,38 +1706,49 @@ for question in questions:
...
 
 ### Parameters
 
 #### vector_column_name: `str`, *Required*
 
+A non-empty string indicating the name of the tensor column to search on.
 
 #### tensor_data: `list/np.ndarray`, *Required*
 
+The query vector data to compare against. This should be provided as a list or a two-dimensional NumPy array of numerical values.
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:1722]: vector data => tensor data.

File: docs/references/pysdk_api_reference.md [hunk 3]
```diff
@@ -1706,38 +1706,49 @@ for question in questions:
...
 
 #### tensor_data_type: `str`, *Required*
 
+Specifies the data type of the tensor. Commonly used types (values) include:
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:1726]: Only support the two types.

File: docs/references/pysdk_api_reference.md [hunk 4]
```diff
@@ -1706,38 +1706,49 @@ for question in questions:
...
+
+- `"float"`
+- `"uint8"`.
+
+#### method_type: `str`, *Required*
+
+A non-empty string indicating the ranking method. Currently, only `"maxsim"` is supported.
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:1733]: What is maxsim?

File: docs/references/pysdk_api_reference.md [hunk 5]
```diff
@@ -1706,38 +1706,49 @@ for question in questions:
...
 
--  `'maxsim'`
+A non-empty string specifying the following search options:
 
-#### extra_option: `str` 
+- **"topn"**: `str`, *Required*  
+  Specifies the number of the most relevant rows to retrieve, e.g., `"topn=10"` to obtain the ten most relevant rows.
+- **EMVB index options**: `str`, *Optional*  
+  Used when an EMVB index is successfully created on the column being queried. Supported options:
+  - `"emvb_centroid_nprobe"`
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:1743]: Each meaning of the parameter.

File: docs/references/pysdk_api_reference.md [hunk 6]
```diff
@@ -1800,10 +1828,10 @@ from infinity.remote_thrift.types import make_match_tensor_expr
...
 ```python
-table_object.fusion('rrf')
-table_object.fusion('rrf', 'topn=10')
-table_object.fusion('weighted_sum', 'weights=1,2,0.5')
-table_object.fusion('match_tensor', 'topn=2', make_match_tensor_expr('t', [[0.0, -10.0, 0.0, 0.7], [9.2, 45.6, -55.8, 3.5]], 'float', 'maxsim'))
+table_object.fusion("rrf")
+table_object.fusion("rrf", "topn=10")
+table_object.fusion("weighted_sum", "weights=1,2,0.5")
+table_object.fusion("match_tensor", "topn=2", make_match_tensor_expr("t", [[0.0, -10.0, 0.0, 0.7], [9.2, 45.6, -55.8, 3.5]], "float", "maxsim"))
 ```
```
Comment (JinHai-CN) [inline docs/references/pysdk_api_reference.md:1835]: What is the meaning of RRF, Weight sum and match tensor.

Comment (JinHai-CN) [reply docs/references/pysdk_api_reference.md:1835]: Need output at the end.

File: docs/references/pysdk_api_reference.md (hunk 4 above)
Comment (yangzq50) [reply docs/references/pysdk_api_reference.md:1733]: https://arxiv.org/abs/2004.12832

File: docs/references/pysdk_api_reference.md (hunk 5 above)
Comment (yangzq50) [reply docs/references/pysdk_api_reference.md:1743]: nprobe: similar to nprobe in IVF
following parameters: chosen in experiment, may vary across different datasets

**Review by yangzq50 — APPROVED** [review]

**Review by JinHai-CN — APPROVED** [review]

**Pull request merged**